infinitest-runner/src/main/java/org/infinitest/config/FileBasedInfinitestConfigurationSource.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Infinitest, a Continuous Test Runner.
+ *
+ * Copyright (C) 2010-2013
```

Comment:
The maven build mandates that the header is exactly the content of the infinitest-parent/COPYING file.

We should probably have a discussion with David and previous maintainer to figure out if we can modify/suppress this header as I don't think it still make sense now that infinitest is MIT license.